conversations/push-ios/remotenotifications/remotenotifications.m

```diff
@@ -1,6 +1,6 @@
 - (void)application:(UIApplication*)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData*)deviceToken {
     if (self.conversationsClient && self.conversationsClient.user) {
-        [self.TwilioConversationsClient registerWithNotificationToken:deviceToken
+        [self.conversationsClient registerWithNotificationToken:deviceToken
```

Comment:
A question here, are we not using anymore TwilioConversationsClient Module? or in this case we just needed to fix it and call the function instead? thanks!